specs/marker/Overview.bs

```diff
@@ -538,7 +538,7 @@ where:
 
 <div class="definition">
   <dfn>&lt;child-selector></dfn> =
-  <div style="margin-left: 4em">select(<a href="http://www.w3.org/TR/selectors4/#compound">compound selector</a>#)</div>
+  <div style="margin-left: 4em">select(<a href="http://www.w3.org/TR/selectors4/#compound">&lt;compound selector&gt;</a> #)</div>
```

Comment:
> I wonder if there are any implementations of it. It might need to get removed all together.

That will be separate issue if tests show a lack of implementations.

Reply:
Given the WG decision to defer the feature, we can still land this fix-up and follow with a PR to delete the feature (and add a Changes entry).